frontend/src/scenes/userLogic.tsx

```diff
@@ -20,7 +21,7 @@ export const userLogic = kea<userLogicType<UserType, UpdateUserPayload>>({
     }),
     reducers: {
         user: [
-            null as UserType | null,
+            (window as any)['POSTHOG_APP_USER'] || (null as UserType | null),
```

Comment:
What's this `POSTHOG_APP_USER`? Leftover code maybe?

Shouldn't the `as UserType | null` apply for the entire statement?

Reply:
Leftover and indeed, the type was broken as well 🤦.